src/plone/api/tests/test_group.py

```diff
@@ -480,17 +481,27 @@ def test_grant_roles_in_context(self):
         ROLES = set(['Editor', 'Contributor'])
         self.assertEqual(
             ROLES,
-            set(api.group.get_roles(
-                groupname='foo', obj=folder, inherit=False)),
+            set(
+                api.group.get_roles(
+                    groupname='foo',
+                    obj=folder,
+                    inherit=False,
+                ),
+            ),
         )
         self.assertEqual(
             ROLES,
-            set(api.group.get_roles(group=group, obj=folder, inherit=False)),
+            set(api.group.get_roles(group=group, obj=folder, inherit=False,)),
```

Comment:
this is even worst; seems to me the flake8 plugin is just broken.

Reply:
`inherit=False,)` should be prohibited not required... I can't repeat this error